plugins/modules/helm.py

```diff
@@ -743,13 +753,14 @@ def main():
                     values_files,
                     chart_version,
                     replace,
+                    chart_repo_url,
                 )
                 if would_change and module._diff:
                     opt_result["diff"] = {"prepared": prepared}
             else:
                 module.warn(
                     "The default idempotency check can fail to report changes in certain cases. "
-                    "Install helm diff for better results."
+                    "Install helm diff >= 3.4.1 for better results."
```

Comment:
Can you please add a note in the docs section about this? So that anyone who reads the docs will get to know.

Reply:
yes I will do it.
Thanks

Reply:
Thanks,